fling/component/client/build_client.py

```diff
@@ -15,3 +15,15 @@ def get_client(
         test_dataset=test_dataset,
         **kwargs
     )
+def get_cross_domain_client(
```

Comment:
Maybe there is no need to add this function. You can use ``**kwargs`` to pass the argument domain.